index.html

```diff
@@ -0,0 +1,205 @@
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 2nd work! -->
+        <div class="works">
+          <img src="images/works/work2.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 3rd work! -->
+        <div class="works">
+          <img src="images/works/work3.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 4rt work! -->
+        <div class="works">
+          <img src="images/works/work4.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+      </section>
+
+      <!-- About me section! -->
+      <section class="aboutme">
+        <div class="title">About Myself</div>
+        <div class="txt">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
```

Comment:
- [ ] Please make sure to wrap this text into a `p` tag, as this is good for semantic purposes as well, so go ahead with those changes!

Reply:
p tag added